[BUG]package "k8s.io/api/core/v1" is being imported more than once

**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

pkg/utils/kubeclient/pod.go:21:2: package "k8s.io/api/core/v1" is being imported more than once (ST1019)
	pkg/utils/kubeclient/pod.go:22:2: other import of "k8s.io/api/core/v1"

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/charts/fluid/fluid/values.yaml
+++ b/charts/fluid/fluid/values.yaml
@@ -4,22 +4,22 @@
 
 dataset:
   controller:
-    image: registry.cn-hangzhou.aliyuncs.com/fluid/dataset-controller:v0.5.0-569c39b
+    image: registry.cn-hangzhou.aliyuncs.com/fluid/dataset-controller:v0.5.0-ebe8884
 
 csi:
   registrar:
     image: registry.cn-hangzhou.aliyuncs.com/acs/csi-node-driver-registrar:v1.2.0
   plugins:
-    image: registry.cn-hangzhou.aliyuncs.com/fluid/fluid-csi:v0.5.0-569c39b
+    image: registry.cn-hangzhou.aliyuncs.com/fluid/fluid-csi:v0.5.0-ebe8884
 
 runtime:
   mountRoot: /runtime-mnt
   alluxio:
     enabled: true
     init:
-      image: registry.cn-hangzhou.aliyuncs.com/fluid/init-users:v0.5.0-569c39b
+      image: registry.cn-hangzhou.aliyuncs.com/fluid/init-users:v0.5.0-ebe8884
     controller:
-      image: registry.cn-hangzhou.aliyuncs.com/fluid/alluxioruntime-controller:v0.5.0-569c39b
+      image: registry.cn-hangzhou.aliyuncs.com/fluid/alluxioruntime-controller:v0.5.0-ebe8884
     runtime:
       image: registry.cn-huhehaote.aliyuncs.com/alluxio/alluxio:2.3.0-SNAPSHOT-985b84c
     fuse:
@@ -29,4 +29,4 @@ runtime:
   jindo:
     enabled: true
     controller:
-      image: registry.cn-hangzhou.aliyuncs.com/fluid/jindoruntime-controller:v0.5.0-569c39b
+      image: registry.cn-hangzhou.aliyuncs.com/fluid/jindoruntime-controller:v0.5.0-ebe8884

--- a/pkg/utils/kubeclient/pod.go
+++ b/pkg/utils/kubeclient/pod.go
@@ -18,15 +18,14 @@ package kubeclient
 import (
 	"context"
 
-	corev1 "k8s.io/api/core/v1"
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
 	apierrs "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // IsCompletePod determines if the pod is complete
-func IsCompletePod(pod *corev1.Pod) bool {
+func IsCompletePod(pod *v1.Pod) bool {
 	if pod == nil {
 		return false
 	}
@@ -35,25 +34,25 @@ func IsCompletePod(pod *corev1.Pod) bool {
 		return true
 	}
 
-	if pod.Status.Phase == corev1.PodSucceeded || pod.Status.Phase == corev1.PodFailed {
+	if pod.Status.Phase == v1.PodSucceeded || pod.Status.Phase == v1.PodFailed {
 		return true
 	}
 	return false
 }
 
 // IsFailedPod determines if the pod is failed
-func IsFailedPod(pod *corev1.Pod) bool {
-	return pod != nil && pod.Status.Phase == corev1.PodFailed
+func IsFailedPod(pod *v1.Pod) bool {
+	return pod != nil && pod.Status.Phase == v1.PodFailed
 }
 
 // GetPodByName gets pod with given name and namespace of the pod.
-func GetPodByName(client client.Client, name, namespace string) (pod *corev1.Pod, err error) {
+func GetPodByName(client client.Client, name, namespace string) (pod *v1.Pod, err error) {
 	key := types.NamespacedName{
 		Name:      name,
 		Namespace: namespace,
 	}
 
-	pod = &corev1.Pod{}
+	pod = &v1.Pod{}
 
 	if err = client.Get(context.TODO(), key, pod); err != nil {
 		if apierrs.IsNotFound(err) {
@@ -67,7 +66,7 @@ func GetPodByName(client client.Client, name, namespace string) (pod *corev1.Pod
 }
 
 //DeletePod deletes the given pod if it exists
-func DeletePod(client client.Client, pod *corev1.Pod) error {
+func DeletePod(client client.Client, pod *v1.Pod) error {
 	err := client.Delete(context.TODO(), pod)
 	if apierrs.IsNotFound(err) {
 		err = nil

